build.sbt

```diff
@@ -5,13 +5,15 @@ import scala.util.parsing.json.JSON
 
 name := """bitbucket-scala-client"""
 
-scalaVersion := "2.11.12"
+scalaVersion := crossScalaVersions.value(1)
+
+crossScalaVersions := Seq("2.11.12", "2.12.10")
```

Comment:
Why don't you create two `val`s with the two versions? Accessing by index can be changed without intent if someone is not careful enough.
For example:
```scala
val scala211 = "2.11.12"
val scala212 = "2.12.10"

scalaversion := scala212
crossScalaVersions := Seq(scala211, scala212)
```